lib/vagrant-kaigara/provisioner.rb

```diff
@@ -1,26 +1,65 @@
 module VagrantPlugins
   module Kaigara
     class Provisioner < Vagrant.plugin(2, :provisioner)
+      def initialize(machine, opts)
+        @machine = machine
+      end
+
       def provision
-        puts "Installing RVM..."
-        script = %{
-          apt-get install curl
-          echo 'export rvm_prefix="$HOME"' > /root/.rvmrc
-          echo 'export rvm_path="$HOME/.rvm"' >> /root/.rvmrc
+        if rvm_installed?
+          @machine.ui.info("Kaigara is already installed")
+        else
+          @machine.ui.info("Installing RVM...")
+
+          install_rvm = %{
+          sudo apt-get install curl
+          echo 'export rvm_prefix=\"$HOME\"' > /root/.rvmrc
```

Comment:
there is no sense to make a root user install.
make a global system install and make sure it's available for all users.